Formula/perl.rb

```diff
@@ -43,6 +30,10 @@ def install
       end
     end
 
+    # We don't want the build process to go snooping around externally
+    # for things like the `Test` perl module.
+    ENV.delete("PERL5LIB")
```

Comment:
I think this can wait for the ENV filtering. I don't think we need to pepper formulae with it.